prac_05/word_occurences.py

```diff
@@ -0,0 +1,45 @@
+"""
+- Program to count the occurrences of words in a string.
+- Asks for a string, prints the counts of how many each
+word in the string.
+- this program is case-insensitive
+
+estimate: 1hr
+actual: 1hr  4mins
+
+"""
+
+
+
+def main():
+    # get input string, convert to list
+    string = input("Enter Text\n>>>")
+    words = string.split()
+    # get longest word length
+    longest_length = find_long_word(words)
+    words_to_count = {}  # pre-allocate
+    # from the slides
+    for word in words:
+        word = word.lower()
+        if word in words_to_count:
+            words_to_count[word] += 1
+        else:
+            words_to_count[word] = 1
+
+    # print results
```

Comment:
This is a good example of a noise comment, you will lose marks if you have too many of these.